buidler-app/.solcover.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Add files to support coverage once we release `test-helpers` compatible with web3 1.x